src/dom.js

```diff
@@ -576,7 +576,7 @@ var dom = module.exports = {
 };
 
 // override setInnerHtml for Windows 8 HTML applications
-if (typeof global.MSApp !== 'undefined') {
+if (typeof global.MSApp !== 'undefined' && global.MSApp.execUnsafeLocalFunction !== 'undefined') {
```

Comment:
We are missing a `typeof` here. I saw this right when I clicked "merge" :disappointed: Going to revert this...